stubs/pywin32/_win32typing.pyi

```diff
@@ -3741,13 +3769,32 @@ class PyIKnownFolderManager:
     def Redirect(self, _id: PyIID, hwnd: int, flags, TargetPath, Exclusion: tuple[PyIID, ...], /) -> None: ...
 
 class PyILockBytes:
-    def ReadAt(self, ulOffset: ULARGE_INTEGER, cb, /) -> str: ...
-    def WriteAt(self, ulOffset: ULARGE_INTEGER, data: str, /): ...
+    @overload
+    @deprecated("Support for passing 2 integers to create a 64bit value is deprecated - pass an int instead")
+    def ReadAt(self, ulOffset: tuple[int, int], cb, /) -> str: ...
+    @overload
+    def ReadAt(self, ulOffset: int, cb, /) -> str: ...
+    @overload
+    @deprecated("Support for passing 2 integers to create a 64bit value is deprecated - pass an int instead")
+    def WriteAt(self, ulOffset: tuple[int, int], data: str, /): ...
+    @overload
+    def WriteAt(self, ulOffset: int, data: str, /): ...
     def Flush(self) -> None: ...
-    def SetSize(self, cb: ULARGE_INTEGER, /) -> None: ...
-    def LockRegion(self, libOffset: ULARGE_INTEGER, cb: ULARGE_INTEGER, dwLockType, /) -> None: ...
-    def UnlockRegion(self, libOffset: ULARGE_INTEGER, cb: ULARGE_INTEGER, dwLockType, /) -> None: ...
-    def Stat(self, grfStatFlag, /) -> STATSTG: ...
+    @overload
+    @deprecated("Support for passing 2 integers to create a 64bit value is deprecated - pass an int instead")
+    def SetSize(self, cb: tuple[int, int], /) -> None: ...
+    @overload
+    def SetSize(self, cb: int, /) -> None: ...
+    @overload
+    @deprecated("Support for passing 2 integers to create a 64bit value is deprecated - pass an int instead")
+    def LockRegion(self, libOffset: tuple[int, int], cb: tuple[int, int], dwLockType, /) -> None: ...
+    @overload
+    def LockRegion(self, libOffset: int, cb: int, dwLockType, /) -> None: ...
+    @overload
+    @deprecated("Support for passing 2 integers to create a 64bit value is deprecated - pass an int instead")
```

Comment:
```suggestion
    @deprecated("Support for passing two ints to create a 64-bit value is deprecated; pass a single int instead")
```

Seems like this occurs a bunch more times, please change the remaining ones too

Reply:
Yeah I copied the string from source verbatim. But I don't mind changing it slightly